pyodide-build/setup.py

```diff
@@ -5,6 +5,7 @@
     setuptools.setup(
         entry_points={
             "console_scripts": [
+                "pywasm = pyodide_build.out_of_tree.__main__:main",
```

Comment:
That's a bit confusing with https://github.com/mohanson/pywasm

So I guess it would be a bit difficult to use subcommands here. Sill maybe something like `pyodide-build-wasm` even if it's a bit more verbose? pywasmbuild would also be better. Ideally, it would be to have pyodide in the name but maybe it will be too complex of a name if we include it.

Reply:
Although if you are calling it as `pywasm build` (I missed the subcommand part), build in the name would indeed be a bit redundant.

Maybe we should just call this command `pyodide` and make this the new CLI API?

Reply:
The main distinction in my mind is between in-tree and out-of-tree commands. E.g., `mkpkg` should be a public in-tree command, whereas it makes no sense out of tree. This is an out of tree command. People are likely to only need one CLI as appropriate to whether they are doing an in tree or out of tree build. So I think it makes sense to keep these as separate CLIs.